app/models/broadcast.rb

```diff
@@ -32,4 +34,8 @@ def single_active_announcement_broadcast
 
     errors.add(:base, "You can only have one active announcement broadcast")
   end
+
+  def update_last_active_at
+    self.last_active_at = Time.zone.now
```

Comment:
Should use [`Time.current`](https://apidock.com/rails/v6.0.0/Time/current/class) here